Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/README.md

```diff
@@ -0,0 +1,1910 @@
+| UmbrellaReporting.ActivityDns.returncode | Number | The DNS return code for this request. For more information, see https://support.umbrella.com/hc/en-us/articles/232254248-Common-DNS-return-codes-for-any-DNS-service-and-Umbrella- | 
+| UmbrellaReporting.ActivityDns.blockedapplications.id | Number | ID of the application. | 
+| UmbrellaReporting.ActivityDns.blockedapplications.label | String | Label of the application. | 
+| UmbrellaReporting.ActivityDns.blockedapplications.type | String | Type of the application, NBAR or AVC. | 
+| UmbrellaReporting.ActivityDns.blockedapplications.category.label | String | Label of the application category. | 
+| UmbrellaReporting.ActivityDns.blockedapplications.category.id | Number | ID of the application category. | 
+
+#### Command example for **`traffic_type = dns`** for base command **`umbrella-reporting-activity-get`**
+```!umbrella-reporting-activity-get traffic_type=dns limit=2```
+#### Context Example for **`traffic_type = dns`** for base command **`umbrella-reporting-activity-get`**
+```json
+{
+    "UmbrellaReporting": {
+        "ActivityDns": [
+            {
+                "allapplications": [
+                    {
+                        "category": {
+                            "id": 46,
+                            "label": "Ad Publishing"
+                        },
+                        "id": 46102,
+                        "label": "Google Marketing Platform"
+                    }
+                ],
+                "allowedapplications": [],
+                "blockedapplications": [],
+                "categories": [
+                    {
+                        "deprecated": false,
+                        "id": 27,
+                        "integration": false,
+                        "label": "Advertisements",
+                        "type": "content"
+                    },
+                    {
+                        "deprecated": false,
+                        "id": 148,
+                        "integration": false,
+                        "label": "Application",
+                        "type": "application"
+                    }
+                ],
+                "date": "2022-10-29",
+                "device": {
+                    "id": null
+                },
+                "domain": "stats.g.doubleclick.net",
+                "externalip": "4.4.4.4",
+                "identities": [
+                    {
+                        "deleted": false,
+                        "id": 593805843,
+                        "label": "S\u2019s MacBook Pro",
+                        "type": {
+                            "id": 9,
+                            "label": "Roaming Computers",
+                            "type": "roaming"
+                        }
+                    }
+                ],
+                "internalip": "192.168.0.105",
```

Comment:
```suggestion
                "internalip": "1.1.1.1",
```
Please apply this to all IPs in the readme file.

Reply:
@ShacharKidor modified